.pre-commit-config.yaml

```diff
@@ -51,16 +51,14 @@ repos:
   rev: v1.24.0
   hooks:
   - id: pyupgrade
-- repo: https://github.com/asottile/seed-isort-config
-  rev: v1.9.3
-  hooks:
-  - id: seed-isort-config
-- repo: https://github.com/pre-commit/mirrors-isort
-  rev: v4.3.21
+- repo: https://github.com/timothycrosley/isort
+  rev: 4.3.21-2
   hooks:
   - id: isort
     name: isort except __init__.py
+    'types': [python]
```

Comment:
nit: I think this line is not necessary because the isort hooks defines it too:
https://github.com/timothycrosley/isort/blob/500bafabbd51a6005c11a00c4738a2438990e48a/.pre-commit-hooks.yaml#L6

Reply:
I started without and had to add it otherwise I get:

```
isort except __init__.py.................................................Failed
hookid: isort

Files were modified by this hook. Additional output:

Fixing /home/gbaconnier/.../stock-logistics-warehouse/stock_orderpoint_move_link/i18n/stock_orderpoint_move_link.pot
Fixing /home/gbaconnier/.../stock-logistics-warehouse/stock_location_zone/i18n/stock_location_zone.pot
Skipped 17 files
```

Reply:
The explanation is that `master` has the types declared in the repo, but the latest release doesn't (https://github.com/timothycrosley/isort/blob/cd9c2f6e70196bd429d0179635e0fac2d034ac5e/.pre-commit-hooks.yaml).

Reply:
In such case, you can add a TODO note here indicating that types should be removed after isort is upgraded to a newer version.